tools/SetupFlow/DevHome.SetupFlow/TaskGroups/RepoConfigTaskGroup.cs

```diff
@@ -23,10 +23,11 @@ public class RepoConfigTaskGroup : ISetupTaskGroup
     private readonly IHost _host;
     private readonly Lazy<RepoConfigReviewViewModel> _repoConfigReviewViewModel;
     private readonly Lazy<RepoConfigViewModel> _repoConfigViewModel;
+    private readonly Guid _activityId;
 
     private readonly ISetupFlowStringResource _stringResource;
 
-    public RepoConfigTaskGroup(IHost host, ISetupFlowStringResource stringResource)
+    public RepoConfigTaskGroup(IHost host, ISetupFlowStringResource stringResource, SetupFlowOrchestrator setupFlowOrchestrator)

```

Comment:
Should this be an GUID instead of Orchestrator?

Reply:
Can't be a guid.  The host service does not know how to resolve a guid and crashes.

Reply:
Oh that's the task group, my bad! I thought it was the task. Yes the task group is registered and will not understand how to resolve a GUID. Resolving.